proof/refine/X64/IpcCancel_R.thy

```diff
@@ -1269,13 +1269,13 @@ lemma (in delete_one) suspend_corres:
 
 lemma no_fail_switchFpuOwner[wp]:
   "no_fail \<top> (X64.switchFpuOwner thread cpu)"
-  by (simp add: X64.switchFpuOwner_def Arch.no_fail_machine_op_lift)
+  by (simp add: X64.switchFpuOwner_def X64.no_fail_machine_op_lift)
```

Comment:
Well spotted! Probably someone getting overzealous with adding qualification, there's indeed no need for it.

Reply:
Looks like removing this qualification breaks the X64 proof. I will investigate why.

Reply:
OK, this isn't an issue with ASpec/AInvs, this is someone doing an Arch-dependent proof outside of the Arch context, because why not. Will revert this "fix" and leave it for proper arch-splitting of Refine.